Scripts/BridgeCreater.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BridgeCreater : MonoBehaviour
+{
+    // Start is called before the first frame update
+    [SerializeField] float zOffset = 1f;
+    [SerializeField] float xOffset = 1f;
+    [SerializeField]
+    int winCondition = 26;
+    int friendIndex = 1;
+    [SerializeField]
+    Vector3 target = new Vector3(459f, 3f, -14f);
+    [SerializeField] float speed = 10f;
+    public bool isWon = false;
+    public bool IsWon
+    {
+        get { return isWon; }
+    }
+    public bool isPlaced = false;
+    public bool IsPlaced
+    {
+        get { return isPlaced; }
+    }
+    [SerializeField] float downLength = 5f;
+
+    public void CreateBridge()
+    {
+        if (GameManager.ObjectToFollow.Count - 1 >= winCondition)
+        {
+            isWon = true;
+            FindObjectOfType<GameManager>().GameWon();
+            for (int i = 1; i <= 5; i++)
```

Comment:
5 magic number, neyi ifade ettiği anlaşılmıyor;

örnek:
```suggestion
            int rowCount = 5;
            for (int i = 1; i <= rowCount; i++)
```